packages/DatePicker/stories/DatePicker.stories.js

```diff
@@ -23,9 +23,11 @@ storiesOf("DatePicker", module)
       hasError: boolean("hasError", false),
     });
     return (
-      <Example locale="en" {...datePickerProps()}>
-        <DatePicker.Input {...inputProps()} />
-      </Example>
+      <div style={{ marginTop: "270px" }}>
```

Comment:
Just a suggestion for the future, I like the `css` prop syntax better:
`<div css={`margin-top: 270px;`}>`